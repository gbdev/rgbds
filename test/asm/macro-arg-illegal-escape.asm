MACRO mac
	DEF s EQUS "\#"
	println "{#s:s}"
ENDM
mac \\\"\t\r\0\n\{\}\,\(\)\w\