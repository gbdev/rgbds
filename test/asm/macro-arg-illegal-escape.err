error: Illegal character escape 'w'
    at macro-arg-illegal-escape.asm(5)
error: Illegal character escape '\' at end of input
    at macro-arg-illegal-escape.asm(5)
Assembly aborted with 2 errors!
