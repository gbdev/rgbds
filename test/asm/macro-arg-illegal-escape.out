\\\"\t\r\0\n\{},()w\\
